### PR TITLE
Stylesheet update

### PR DESCRIPTION
**Pull request by solth**
Second stylesheet update pass.

### CODE REVIEW

**Comment by henning-gerhardt** [comment]
There is at least one merge conflict in `Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css`. Please solve this conflict.

File: Kitodo/src/main/webapp/pages/kitodo-frontend/firstpage.xhtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!--
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/kitodo-frontend/firstpage.xhtml]: What is the reason to add this and the other file `secondpage.xhtml`?

File: Kitodo/src/main/webapp/pages/kitodo-frontend/test/secondpage.xhtml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/kitodo-frontend/test/secondpage.xhtml]: What is the reason to add this and the other file `firstpage.xhtml`?

**Comment by solth** [comment]
> See my comment to `firstpage.xhtml` and `secondpage.xhtml`. I do not think that this files should be added or?

You are right, these were not supposed to be added with this commit. Therefore I removed them again. Thanks for pointing it out!

**Comment by solth** [comment]
> There is at least one merge conflict in Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css. Please solve this conflict.

Rebased and conflict resolved.

**Pull request merged**